src/plugins/plot/src/configuration/YAxisModel.js

```diff
@@ -146,10 +146,14 @@ define([
                     this.updateStats(seriesStats);
                 }
             }, this);
+            this.listenTo(series, 'change:yKey', function () {
+                this.updateFromSeries(this.seriesCollection);
+            }, this);
         },
         untrackSeries: function (series) {
             this.stopListening(series);
             this.resetStats();
+            this.updateFromSeries(this.seriesCollection);
```

Comment:
#2029 might be relative to fix the issue it is closing. `this.updateFromSeries` doesn't get called when telemetry is empty / table is empty. The function sets defaults that are used later for empty telemetry tables.